### ISSUE TITLE
Error on editing Activity

### ISSUE THREAD

**Issue by splattner** [unfoldedcircle/bug-reports#38]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

I added a new Activity but did an error on button mapping. So I went back to change a button to an other commad. But now, it seems the change is not working anymore. The webconfigurator show the correct mapping. But when hitting the button on the remote, nothing happens.

When I switch back from the "Edit User Interface" I get the following Error in Red and with the following content in the Browser Dev console.

```
error.ts:22 TypeError: Cannot read properties of undefined (reading 'length')
    at SequenceItem.vue:110:29
    at runtime-core.esm-bundler.js:2739:40
    at We (runtime-core.esm-bundler.js:157:22)
    at ve (runtime-core.esm-bundler.js:166:21)
    at t.__weh.t.__weh (runtime-core.esm-bundler.js:2713:29)
    at Nn (runtime-core.esm-bundler.js:343:46)
    at Di (runtime-core.esm-bundler.js:397:9)
```

### How to Reproduce

* Add an activity
* Edit the activity again later
* Change a button
* Return from the "edit user interface" page
* Testing the button on the remote

### Expected behavior

To have the changes button woking

### System version

v1.1.2

### What part of the system affected by the problem?

Core, Web Configurator, Overall system

**Comment by Depechie** [comment]
All my activities now show this same error after I upgraded from v1.1.1 to v1.1.2.
Just going to the activity in the web configurator interface.
Did not alter any settings and the activities were correct with v1.1.1

**Comment by Freshhat** [comment]
Seems to be related to my issue #42 i see that i get the same error in the Browser Console when trying to modify/create/clone macros:

```
TypeError: Cannot read properties of undefined (reading 'length')
    at SequenceListFiltered.vue_vue_type_script_setup_true_lang-1f1dbe77.js:1:3314
    at vue-select-611b60d3.js:1:31919
    at We (vue-select-611b60d3.js:1:13749)
    at ve (vue-select-611b60d3.js:1:13827)
    at t.__weh.t.__weh (vue-select-611b60d3.js:1:31799)
    at Nn (vue-select-611b60d3.js:1:15120)
    at Di (vue-select-611b60d3.js:1:15427)
```

**Comment by splattner** [comment]
Update: the problem, that the button was not reacting was probably more due to this https://github.com/unfoldedcircle/bug-reports/issues/34

But still, I get the JS Error, otherwise, I think no impact?

**Comment by zehnm** [comment]
Most of the `Cannot read properties of undefined` in the edit activity & macro screens should be fixed in 1.3.3.
Otherwise please re-open this issue or create a new one.
